src/query/v2/requests.hpp

```diff
@@ -202,9 +418,24 @@ struct ExpandOneResponse {
   std::vector<ExpandOneResultRow> result;
 };
 
+// Update related messages
+struct UpdateVertexProp {
+  VertexId primary_key;
+  std::vector<std::pair<PropertyId, Value>> property_updates;
+};
+
+struct UpdateEdgeProp {
+  Edge edge;
```

Comment:
Why not `EdgeId`?

Reply:
The corresponding handle, that actually interacts with the `Shard` is not yet implemented. It will probably make sense to use just the `EdgeId` instead, however I think that should be done in the PR that will contain the implementation of said handle. I don't think we should set the message's format into stone now because the exact way we might use them can change and we can only be sure once the `Shard`-side handle is in place.